web/src/views/IndividualResults/StudyPage.vue

```diff
@@ -64,6 +65,24 @@ export default defineComponent({
       });
     });
 
+    const goldLinks = computed(() => {
+      if (!item.value) {
+        return [];
+      }
+      const links = new Set();
+      if (item.value.open_in_gold) {
+        links.add(item.value.open_in_gold);
```

Comment:
With the definition of `gold_url` [here](https://github.com/microbiomedata/nmdc-server/blob/cf4015789ce012c7f8f856362e8c08e918b0d846/nmdc_server/models.py#L33-L38), it would seem that when the id does not start with `gold:`, first gold identifier would be added here, then added again from the gold_study_identifiers list below. Are you doing anything to address this so it is not duplicated?

Reply:
Yes, I'm using a [set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/add), so that an attempt to add a duplicate link would not basically be a no-op. When running locally, I've checked study pages with various `id` and `gold_study_identifiers`, and see no duplicate links.

Reply:
If its more clear to use an array and an explicit call to `.includes`, I wouldn't mind changing it. I don't think we get much of a performance gain here from using a set, since this list will probably stay pretty small, but it felt like the right data structure here.

Reply:
Ah, I missed that it is a set. Great makes sense we can keep this as is.